roles/ceph-config/tasks/main.yml

```diff
@@ -12,20 +12,31 @@
       - /etc/ceph/
       - /etc/ceph/ceph.d/
 
+### Work-around for host-specific ceph_config_overrides generation
```

Comment:
please remove this comment

Reply:
@leseb done. Sorry, I've just re-pushed a branch... Should I add fixes as additional commits in a pull request in the future?

Reply:
Update commit's message as well

Reply:
You can keep the same commit if that's fixing the same thing. Thanks.